defaults/main.yml

```diff
@@ -0,0 +1,129 @@
+---
+#General
+  dokuwiki_filename: dokuwiki-c5525093cf2c4f47e2e5d2439fe13964.tgz
```

Comment:
What is this file even for?
Can't you retrieve it from a repository?
Otherwise, just download it, but be sure to use a https link or verify the hashes